Code/LauncherUnified/Launcher.cpp

```diff
@@ -398,6 +398,9 @@ namespace O3DELauncher
             AZ::SettingsRegistryMergeUtils::ProjectSettingsRootKey,
             aznumeric_cast<int>(buildTargetName.size()), buildTargetName.data());
 
+        const bool dedicatedServer = IsDedicatedServer();
+        AZ::SettingsRegistryMergeUtils::MergeSettingsToRegistry_AddClientServerTargetSpecialization(*settingsRegistry, dedicatedServer);
```

Comment:
NP: Not sure if whitespace is weird here but worth a check

Reply:
Ah, good find, evil tabs!

Reply:
Oh, the whitespace is fine the above portion belongs to AZ_TracePrintf input.